Signal/src/ViewControllers/SettingsTableViewController.h

```diff
@@ -19,6 +19,7 @@
 @property (strong, nonatomic) IBOutlet UILabel *linkedDevicesLabel;
 @property (strong, nonatomic) IBOutlet UILabel *advancedLabel;
 @property (strong, nonatomic) IBOutlet UILabel *aboutLabel;
+@property (strong, nonatomic) IBOutlet UILabel *blocklistLabel;
```

Comment:
nit: sorry to be pedantic, but it seems like we're elsewhere treating block-list as *two* words, so I guess this should be blockListLabel.